backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/flightleg/BpkFlightLeg.kt

```diff
@@ -73,13 +77,19 @@ fun BpkFlightLeg(
     ) {
         Row(modifier = Modifier.fillMaxWidth()) {
             carrierLogoContent?.let {
-                Box(
-                    modifier = Modifier.padding(top = BpkSpacing.Sm),
-                    content = it,
-                )
+                Box(modifier = Modifier.padding(top = BpkSpacing.Sm)) {
+                    Box(
+                        modifier = Modifier
+                            .size(BpkSpacing.Lg)
+                            .background(color = BpkTheme.colors.textOnDark, shape = RoundedCornerShape(BpkBorderRadius.Xs)),
```

Comment:
The rounded corners don't work if you use `carrierLogoContent = { Box(Modifier.fillMaxSize().background(Color.Red)) }`

You need to it differently:

```
.clip(RoundedCornerShape(BpkBorderRadius.Xs))
.background(BpkTheme.colors.textOnDark)
```

Reply:
thanks applied